### PR TITLE
Separated panel component from grid component

### PR DESCRIPTION
**Pull request by TannerHolladay**

### CODE REVIEW

File: client/src/components/molecules/Grid/Grid.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  return (
+    <GridLayout
+      className="layout"
+      breakpoints={{ lg: 1200, md: 996, sm: 768, xs: 480, xxs: 0 }}
+      cols={{ lg: 3, md: 3, sm: 3, xs: 3, xxs: 3 }}
+      rowHeight={rowHeight}
+      isResizable={false}
+      margin={[15, 15]}
+      containerPadding={[20, 20]}
```
Comment (sethduffin) [inline client/src/components/molecules/Grid/Grid.tsx:21]: Set padding to `[0, 0]` here and move the padding to the Panel component

File: client/src/components/molecules/GridPanel/Panel.scss [hunk 2]
```diff
@@ -0,0 +1,123 @@
+.panel {
+  position: relative;
+  background: #f3f4f6;
+  border-radius: 10px;
+  width: 100%;
+  overflow: hidden;
```
Comment (sethduffin) [inline client/src/components/molecules/GridPanel/Panel.scss:6]: Remove `overflow: hidden;` to allow the pop-out menu to show when panel is collapsed

File: client/src/components/molecules/GridPanel/Panel.scss [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  }
+
+  &.open {
+    .fa-caret-down {
+      transform: rotateZ(0deg);
+      animation: rotate-down 0.2s ease;
+    }
+  }
+
+  &__content {
```
Comment (sethduffin) [inline client/src/components/molecules/GridPanel/Panel.scss:60]: Insert `overflow: hidden;` here

File: client/src/components/molecules/GridPanel/Panel.scss [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    > div {
+      margin: 0 20px 20px;
```
Comment (sethduffin) [inline client/src/components/molecules/GridPanel/Panel.scss:62]: Replace `margin: 0 20px 20px;` with `padding: 16px;`

File: client/src/components/organisms/GridContainer/GridContainer.scss [hunk 5]
```diff
@@ -1,130 +1,9 @@
...
-
-@keyframes rotate-down {
-  from {
-    transform: rotateZ(-90deg);
-  }
-  to {
-    transform: rotateZ(0deg);
-  }
-}
+  padding-bottom: 10px;
```
Comment (sethduffin) [inline client/src/components/organisms/GridContainer/GridContainer.scss:8]: Move `padding-bottom: 10px;` inside of `.react-grid-layout {` to allow the panel to collapse completly

**Review by sethduffin — COMMENTED** [review]
Looks good, just a couple of minor things.
* It would make more sense to rename the `PanelGrid` folder to just `Panel` to match the component filename
* The panel container has to allow overflow so the pop-out menu is visible when the panel is collapsed
*The padding should be in one place on the div inside the `panel_content` container so the panel can collapse completely

File: client/src/components/molecules/Grid/Grid.tsx (hunk 1 above)
Comment (TannerHolladay) [reply client/src/components/molecules/Grid/Grid.tsx]: Oh, good catch. I forgot I added this and was wondering why there was padding

File: client/src/components/molecules/GridPanel/Panel.scss (hunk 2 above)
Comment (TannerHolladay) [reply client/src/components/molecules/GridPanel/Panel.scss]: Ahh, I see. I changed it to this to make it so you can see the tile when you drag it above the contents area, but this is much better to have.

**Review by sethduffin — APPROVED** [review]

**Pull request merged**